demo/index.html

```diff
@@ -146,7 +145,7 @@
   <h3>Styling and positioning</h3>
   <demo-snippet>
     <template>
-      <style is="custom-style">
+      <custom-style><style is="custom-style">
         paper-dialog.colored {
           border: 2px solid;
           border-color: var(--paper-green-500);
```

Comment:
I don't think the colors are really considered part of the external API of demo-pages-shared-styles.